server/entity/arrow.go

```diff
@@ -222,10 +260,11 @@ func (a *Arrow) ignores(entity world.Entity) bool {
 }
 
 // New creates and returns an Arrow with the position, velocity, yaw, and pitch provided. It doesn't spawn the Arrow
-// by itself.
-func (a *Arrow) New(pos, vel mgl64.Vec3, yaw, pitch float64, owner world.Entity, critical, disallowPickup, obtainArrowOnPickup bool, tip potion.Potion) world.Entity {
-	arrow := NewTippedArrow(pos, yaw, pitch, owner, tip)
+// by itself. The second return value is true if an Arrow should be consumed.
```

Comment:
This documentation isn't correct anymore either.

Reply:
Removed.